package/x86_64/macOS_10.12/package.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
```

Comment:
This script is the same for Ubuntu as well. The only difference is we need to run it on host.